open_seq2seq/models/model.py

```diff
@@ -168,6 +169,8 @@ class docs.
     * **summaries** (list) --- which summaries to log. Could contain
       "learning_rate", "gradients", "gradient_norm", "global_gradient_norm",
       "variables", "variable_norm".
+    * **iter_size** (int) --- same as in nvcaffe, the gradients will be
```

Comment:
lets remove nvcaffe reference: 1) it is not technically same due to fp32 aggregation and 2) same as in bla not a very good style for documentation

Reply:
Maybe add more about the fact that it can be used to "simulate" larger batches